src/ports/postgres/modules/utilities/validate_args.py_in

```diff
@@ -344,6 +344,50 @@ def get_cols_and_types(tbl):
     return list(zip(col_names, col_types))
 # -------------------------------------------------------------------------
 
+def get_col_type(tbl, col):
```

Comment:
There's lots of code overlap with `get_cols_and_types`. We should avoid that redundancy. 
Also, does `get_expr_type` not work for this need?